src/renderer/src/components/playlist/playlist-title/PlaylistTitle.tsx

```diff
@@ -0,0 +1,25 @@
+import { PlaylistPage, setPlaylistActivePage } from "../playlist.utils";
+import Button from "@renderer/components/button/Button";
+import { ArrowLeftIcon } from "lucide-solid";
+import { Component } from "solid-js";
+
+type PlaylistTitleProps = {
+  title: string;
+  backTo?: PlaylistPage;
+};
+
+export const PlaylistTitle: Component<PlaylistTitleProps> = (props) => {
+  return (
+    <div class="flex w-full flex-row items-center gap-2.5 py-1 font-medium">
+      <Button
+        variant="ghost"
+        size="square"
+        type="button"
+        onClick={() => setPlaylistActivePage(props.backTo ?? { name: "list" })}
+      >
+        <ArrowLeftIcon class="text-subtext" size={20} />
+      </Button>
+      <h3 class="text-xl uppercase">{props.title}</h3>
```

Comment:
I don't really like the title being in uppercase. Do you think it's ok? (sorry i don't know much about design)

Reply:
We are using upper case on the title to have hierarchical clarity on subpages, uppercase text can differentiate titles or headers from body text, aiding visual hierarchy.

Reply:
Interesting, thanks for the explanation!